src/routes/Home.tsx

```diff
@@ -158,12 +163,25 @@ const Home = () => {
     }),
     {
       input: {
-        projects: loadedProjects,
+        projects: projectPaths,
         defaultProjectName: settings.projects.defaultProjectName.current,
         defaultDirectory: settings.app.projectDirectory.current,
       },
     }
   )
+
+  useEffect(() => {
+    send({ type: 'Read projects', data: {} })
+  }, [projectPaths])
```

Comment:
It will be clearer if I explain this:

This effect exists because we need a way to also tell the state machine to update its internal projects listing.